standard/expressions.md

```diff
@@ -5984,6 +6069,27 @@ When a property or indexer declared in a *struct_type* is the target of an assig
 >
 > *end example*
 
+### §deconstructing-assignment-new-clause Deconstructing assignment
+
+If the left operand of a `=` operator is classified as a tuple, the right hand side is ***deconstructed*** into individual expressions, which are assigned to each of the elements of the left hand side tuple.
+
+If any of the tuple elements of the left operand have an identifier, a compile-time error occurs. If any of the tuple elements of the left operand is a declaration expression, and any other element is not a declaration expression or discard, a compile-time error occurs.
```

Comment:
```suggestion
If any of the tuple elements of the left operand has an identifier, a compile-time error occurs. If any of the tuple elements of the left operand is a declaration expression, and any other element is not a declaration expression or discard, a compile-time error occurs.
```

(Just to be consistent in terms of singular or plural.)

Reply:
I think an example showing both of these would be useful though. In my brain-fogged state, I don't fully understand the first constraint.

Or possibly I don't understand why tuple elements are used. Where do we prevent this from being valid, for example:

```csharp
(1, 2) = (3, 4);
```

Reply:
The singular/plural is fixed in upcoming commit. As for legality of the LHS, this is clarified in the new verbiage, in a paragraph that (recursively if necessary) either finds the type of a simple assignment or deems it invalid.